frontends/bmd/src/pages/poll_worker_screen.tsx

```diff
@@ -152,14 +149,42 @@ function PrecinctScannerTallyReportModal({
     }
   }
 
+  const currentTime = Date.now();
+
+  async function printReport(copies: number) {
+    const signedQuickResultsReportingUrl =
+      await getSignedQuickResultsReportingUrl({
+        electionDefinition,
+        isLiveMode: precinctScannerTally.isLiveMode,
+        compressedTally: precinctScannerTallyInformation.overallTally,
+        signingMachineId: machineConfig.machineId,
+      });
+
+    await printElement(
+      <PrecinctScannerFullReport
+        electionDefinition={electionDefinition}
+        precinctSelectionList={precinctScannerTallyInformation.precinctList}
+        subTallies={precinctScannerTallyInformation.subTallies}
+        isPollsOpen={precinctScannerTally.isPollsOpen}
+        isLiveMode={precinctScannerTally.isLiveMode}
+        currentTime={currentTime}
+        pollsToggledTime={precinctScannerTally.timePollsToggled}
+        totalBallotsScanned={precinctScannerTally.totalBallotsScanned}
+        precinctScannerMachineId={precinctScannerTally.machineId}
+        signedQuickResultsReportingUrl={signedQuickResultsReportingUrl}
+      />,
+      {
+        sides: 'one-sided',
+        copies,
+      }
+    );
+    await sleep(REPORT_PRINTING_TIMEOUT_SECONDS * 1000);
```

Comment:
Hey, I think we can! I see that as a part of #1531. The question is whether it will be fast enough. We can get print job ids and poll `lpserver` for prints being done, I've played with this a bit on linux. It seems like it would work but the print job shows as "complete" in lpserver significantly after it _actually_ completes. Maybe that was just a lag in the UI I was using? Anyway - I think some version of it is possible, and I plan to explore it as I take on #1531 after Polls Suspension.

Reply:
Cool!!